src/XMakeBuildEngine/Evaluation/ProjectParser.cs

```diff
@@ -186,6 +187,62 @@ private void ParseProjectElement(XmlElementWithLocation element)
             // so we have to set it now
             _project.SetProjectRootElementFromParser(element, _project);
 
+            if (element.HasAttribute(XMakeAttributes.sdk))
+            {
+                // TODO: don't get root of SDKs from the environment, use a built-in or toolset prop
+                // TODO: Don't null ref if MSBuildSDKsPath isn't defined
+
+                var sdksString = element.GetAttribute(XMakeAttributes.sdk);
+
+                var sdks = sdksString.Contains(";") ? sdksString.Split(';') : new[] {sdksString};
```

Comment:
I'd use `.Split(';', RemoveEmptyEntries).Select(i => i.Trim()).Where(i => !String.IsNullOrWhiteSpace())` to be as robust as possible.